apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerTransaction.cs

```diff
@@ -315,10 +315,15 @@ Task<long> ISpannerTransaction.ExecuteDmlAsync(ExecuteSqlRequest request, Cancel
                 var callSettings = SpannerConnection.CreateCallSettings(settings => settings.ExecuteStreamingSqlSettings, timeoutSeconds, cancellationToken);
                 using (var reader = _session.ExecuteSqlStreamReader(request, callSettings))
                 {
-                    Value value = await reader.NextAsync(cancellationToken).ConfigureAwait(false);
-                    if (value != null)
+                    var rowsAffected = 0;
+                    while ((await reader.NextAsync(cancellationToken).ConfigureAwait(false)) != null)
```

Comment:
I'd assume the stats are still available - I don't think we should bother reading through all the results if we haven't been asked to do so.